tools/cmdr.py

```diff
@@ -145,12 +196,14 @@ def main():
         "EnableRemoteDesktop",
         "DisableRemoteDesktop",
         "ActivationLockBypassCode",
+        "ScheduleOSUpdate",
```

Comment:
No need to put this here as ScheduleOSUpdate has its own parser. `simple_command_subparser` is for commands that don't need/require subparsers. :)